tutorial/cloudshell_tutorial.md

```diff
@@ -111,9 +109,11 @@ commands.
 
 Run the command `terraform init` in the same directory as your main.tf file now:
 
-```raw
```

Comment:
For these command+output pairs, I'd split it into a runnable command + a sample output.